docs/contributing/source/ways-can-contribute/vanshdeep-notes.md

```diff
@@ -0,0 +1,163 @@
+# Vanshdeep's Notes on Running a Local Dev Node
+
+The following doc describes how to run a local node for developing BigchainDB Tendermint version.
+
+There are two crucial dependencies required to start a local node:
+
+- MongoDB
+- Tendermint
+
+and of course you also need to install BigchainDB Sever from the local code you just developed.
+
+
+## Installing MongoDB
+
+MongoDB can be easily installed, just refer their [installation documentation](https://docs.mongodb.com/manual/installation/) for your distro. 
+We know MongoDB 3.4 works with BigchainDB.
+MongoDB 3.6 _might_ work, or it might not. You could try it.
+After the installation of MongoDB is complete, run MongoDB using `sudo mongod --replSet=bigchain-rs`
```

Comment:
Remove `--replSet=bigchain-rs`, we don't use it anymore.

Suggestion: I'd drop this sentence. People can run `mongod` in many different ways. I personally like to run it as a normal user writing data somewhere in my home directory.

Reply:
@kansi Is this right? I haven't tried it lately.

Reply:
@ttmc Yeah, you can remove it. I just grep the shell command from my history to run mongod, so it uses 1.x args.